packages/design-system/src/styles/settings/uswds/_index.scss

```diff
@@ -0,0 +1,81 @@
+/*
+* * * * * ==============================
+* * * * * ==============================
+* * * * * ==============================
+* * * * * ==============================
+========================================
+========================================
+========================================
+*/
+
+// -------------------------------------
+// Import individual theme settings
+
+@import 'uswds-theme-general';
+@import 'uswds-theme-typography';
+@import 'uswds-theme-spacing';
+@import 'uswds-theme-color';
+@import 'uswds-theme-utilities';
```

Comment:
should we comment out the other imports other than spacing since we are not using colors, or typography, etc...